smart-metastore/src/main/java/org/smartdata/metastore/MetaStore.java

```diff
@@ -1117,6 +1119,22 @@ public void insertAction(ActionInfo actionInfo)
     }
   }
 
+  public List<ActionInfo> searchAction(String path, long start,
+                                       long offset, List<String> orderBy,
+                                       List<Boolean> desc) throws MetaStoreException {
+    try {
+      if (orderBy.size() == 0) {
+        return actionDao.searchAction(path, start, offset);
```

Comment:
Call one function instead of two. The arguments can be handled in the function internally.